scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -119,6 +118,10 @@ function get_config() {
         iniGet "image_delay"
         IMAGE_DELAY="$ini_value"
         [[ -z "$IMAGE_DELAY" ]] && IMAGE_DELAY=2
+        iniGet "log_dir"
+        LOGDIR="$ini_value"
+        [[ -z "$LOGDIR" ]] && LOGDIR="/dev/shm"
```

Comment:
Please modify `[[ -z "$LOGDIR" ]] && LOGDIR="/dev/shm"` to `[[ -d "$LOGDIR" ]] || LOGDIR="/dev/shm"`.

If the `$LOGDIR` is unset or not a directory, default to `/dev/shm`. Though mind `/dev/shm` isn't guaranteed to be mounted. `/tmp` would be the safest bet.